src/main/java/org/terasology/miniion/componentsystem/controllers/CharacterMovementSystem.java

```diff
@@ -55,6 +55,7 @@ public void update(float delta) {
                     characterMovementComponent.directionToMove,
                     characterMovementComponent.runningRequested,
                     characterMovementComponent.jumpingRequested, (long) delta));
+            if(characterMovementComponent.jumpingRequested) characterMovementComponent.jumpingRequested = false;
```

Comment:
This looks like a valid change, but it would be best to surround the statement(s) after the "if" condition with curly braces {}